### PR TITLE
Home

### PR DESCRIPTION
**Pull request by alejo9226**
### Add home, login and register routes

Add home, login and register routes and their components completely styled. Also added requests to register component to create admins into the database

Screen Shot 2020-11-21 at 17 22 22
Screen Shot 2020-11-21 at 17 22 38
Screen Shot 2020-11-21 at 17 22 52
Screen Shot 2020-11-21 at 17 24 54
Screen Shot 2020-11-21 at 17 26 25

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -13,12 +19,56 @@ const theme = {
...
+      password: '',
+      message: '',
+    }
+  };
+
+  handleChange = (e) => {
+    const { name, value, checked, type } = e.target
+
+    this.setState({ login: { ...this.state.login, [name]: value }}, () => console.log(this.state))
+    // this.setState({ [e.target.name]: e.target.value })
```
Comment (dafegosa) [inline src/App.js]: Quitar los comentarios

File: src/App.js [hunk 2]
```diff
@@ -13,12 +19,56 @@ const theme = {
...
+  }
+
+  handleSubmit (e) {
+    e.preventDefault()
+    // console.log(this.state)
```
Comment (dafegosa) [inline src/App.js]: Quitar los comentarios

File: src/App.js [hunk 3]
```diff
@@ -13,12 +19,56 @@ const theme = {
...
+
+    if (isFound.length > 0) {
+      if (isFound[0].password === password) {
+        this.props.history.push('/dashboard')
+      } else {
+        this.setState({...this.props.user, message: 'Wrong password'})
+      }
+    }
+
+    /* this.setState({
```
Comment (dafegosa) [inline src/App.js]: Igual, todo esto está comentado?

File: src/pages/register/Register.js [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+  };
+
+  handleInputChange = (e) => {
+    const { name, value } = e.target;
+    this.setState({ [name]: value }, console.log(this.state));
+  };
+
+  createUser = async (e) => {
+    e.preventDefault();
+    console.log(e.target)
```
Comment (dafegosa) [inline src/pages/register/Register.js]: Quitar los console.log

File: src/pages/register/Register.js [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+    const { name, lastname, idnumber, phone, email, password } = this.state
+    const newUser = {
+      name,
+      lastName: lastname,
+      idNumber: idnumber,
+      phone,
+      email,
+      password,
+    }
+    console.log(newUser)
```
Comment (dafegosa) [inline src/pages/register/Register.js:47]: Quitar console.log

File: src/pages/register/Register.js [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+    try {
+      const { data } = await axios({
+        method: 'POST',
+        baseURL: 'http://localhost:8000',
+        url: '/admin',
+        data: newUser
+      })
+  
+      console.log(data.message);
```
Comment (dafegosa) [inline src/pages/register/Register.js]: console.log

File: src/pages/register/Register.js [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+      this.setState({ ...this.state, message: data.message });
+    } catch(err)  {
+      console.log(err)
```
Comment (dafegosa) [inline src/pages/register/Register.js]: console.log

**Review by dafegosa — CHANGES_REQUESTED** [review]

**Review by dafegosa — APPROVED** [review]

**Review by simonhoyos — APPROVED** [review]

**Pull request merged**